curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/671f774373f2ce0696fe9621.md

```diff
@@ -0,0 +1,92 @@
+---
+id: 671f774373f2ce0696fe9621
+title: Task 100
+challengeType: 22
+dashedName: task-100
+---
+
+<!-- (Audio) Brian: Speaking of which, did you manage to catch the cybersecurity talk this morning? -->
+
+# --instructions--
+
+Listen to the audio and complete the sentence below.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`Speaking of which, did you BLANK to catch the BLANK talk this morning?`
+
+## --blanks--
+
+`manage`
+
+### --feedback--
+
+This word means to succeed in doing something, often with some difficulty.
+
+---
+
+`cybersecurity`
+
+### --feedback--
+
+This word refers to the practice of protecting systems, networks, and data from digital threats.
+
+# --explanation--
+
+The phrase `speaking of which` is used to link a new topic to something that was just mentioned, often as a natural transition in conversation. Example: `The keynote speaker mentioned Python. Speaking of which, have you tried the new Python library?` (here, `speaking of which` is used to make the transition between the mention the keynote speaker made and the new library) 
+
+In this context, `manage` means to succeed in doing something, especially something that might have been difficult. Example: `Despite the time crunch, I managed to submit the report on time.` (this means you were successful in sending the report before your deadline) 
+
+`Cybersecurity` refers to the practice of protecting computer systems, networks, and data from digital attacks or unauthorized access. Example: `The company invested in cybersecurity measures to prevent data breaches.` (this means the company is concerned with protecting their systems) 
+
+In this sentence, Brian asks if Maria attended the talk about protecting systems and data from threats in the morning.
```

Comment:
```suggestion
`Speaking of which` is used to link a new topic to something that was just mentioned, often as a natural transition in conversation. For example: 

`The keynote speaker mentioned Python. Speaking of which, have you tried the new Python library?` - here, `speaking of which` is used to make the transition between the mention the keynote speaker made and the new library. 

In this context, `manage` means to succeed in doing something, especially something that might have been difficult. For example: 

`Despite the time crunch, I managed to submit the report on time.` - This means you were successful in sending the report before your deadline. 

`Cybersecurity` refers to the practice of protecting computer systems, networks, and data from digital attacks or unauthorized access. For example: 

`The company invested in cybersecurity measures to prevent data breaches.` - This means the company is concerned with protecting their systems. 

In this sentence, Brian asks if Maria attended the talk about protecting systems and data from threats in the morning.
```